doc/monitoring/getting_started.rst

```diff
@@ -63,6 +57,55 @@ Now you can use the HTTP API endpoint ``/metrics`` to collect your metrics
 in the Prometheus format. If you need your custom metrics, see the
 :ref:`API reference <metrics-api-reference>`.
 
+.. _metrics-http
+-------------------------------------------------------------------------------
+Collect HTTP metrics
+-------------------------------------------------------------------------------
+
+To enable collection of HTTP metrics, you need to create collector
+
+Using HTTP v1:
+
+.. code-block:: lua
+
+    local httpd = require('http.server').new(ip, port)
+
+    -- Create summary latency collector
+    local collector = metrics.http_middleware.build_default_collector('summary')
+
+    -- Set route handler with summary latency collection
+    httpd:route({ path = '/path-1', method = 'POST' }, metrics.http_middleware.v1(handler_1, collector))
```

Comment:
Well, since we recommend to use the only one collector, maybe it is better to give an example with `configure_default_collector` as a basic approach and do not pass any arguments explicitly?

Reply:
default_collector in `configure_default_collector` is `histogram`

Reply:
`build_default_collector` is a confusing name itself, since it is a plain `build_collector` and do not have any deal with setting up defaults. This code is confusing as well. I think we shouldn't use `build_default_collector` in examples and should recommend the following approach: if you do not want to use default histogram collector, use `configure_default_collector('summary', params)` in your code.

Reply:
Hi @yngvar-antonsson, @DifferentialOrange. I am in charge of proofreading and localizing Tarantool docs. Kindly let me know whether you're going to introduce more changes to the metrics documentation. Thank you.

Reply:
Hi folks. Good enough docs right now are better than perfect docs someday. Let's just merge it as is and make one more iteration someday later.